NetcoreBootstrap.Tests/Controllers/api/v1/HomeApiTestController.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using Xunit;
+using NetcoreBootstrap;
+using System.Net.Http;
+using NetCoreBootstrap;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Configuration;
+using System.IO;
+
+namespace NetcoreBootstrap.Tests.Controllers.api.v1
+{
+    public class HomeApiTestController
+    {
+        private readonly TestServer _server;
+        private readonly HttpClient _client;
+
+        public HomeApiTestController()
+        {
+            var configuration = new ConfigurationBuilder()
+                                .SetBasePath(Path.GetFullPath(@"../../../../NetcoreBootstrap/"))
+                                .AddJsonFile("appsettings.Development.json", optional: false)
+                                .Build();
+            this._server = new TestServer(new WebHostBuilder()
+                            .UseStartup<Startup>()
+                            .UseConfiguration(configuration));
+            this._client = this._server.CreateClient();
+        }        
+
+        public HttpClient Client
+        {
+            get { return this._client; }
+        }
+
+        [Fact]
+        public async Task Get()
+        {
+            // Act
+            var response = await Client.GetAsync("/api/v1/homeapi/1");
```

Comment:
Hmmm, we need to change this `1` for a random number

Reply:
Done! I also changed the controller so that it returns the value it gets from the URL